src/main/java/de/zalando/zmon/scheduler/ng/config/RestTemplateConfiguration.java

```diff
@@ -57,6 +59,7 @@ public RestTemplate restTemplate(ClientHttpRequestFactory clientFactory, TokenWr
 
         MappingJackson2HttpMessageConverter converter = new MappingJackson2HttpMessageConverter();
         converter.setObjectMapper(mapper);
+        converter.setSupportedMediaTypes(Collections.singletonList(MediaType.ALL));
```

Comment:
needed because zmon-controller doesn't set content type on `EntityApi.getEntity` (like it does for `getEntities`)